test/test_runner/base_test_runner.go

```diff
@@ -31,6 +34,8 @@ type ITestRunner interface {
 	GetMeasuredMetrics() []string
 	SetupBeforeAgentRun() error
 	SetupAfterAgentRun() error
+	SSM() bool
```

Comment:
nit - I can't tell what this means. 
```suggestion
	UseSSM() bool
```